diff --git a/node_modules/@gorhom/bottom-sheet/lib/commonjs/components/bottomSheetTextInput/BottomSheetTextInput.js b/node_modules/@gorhom/bottom-sheet/lib/commonjs/components/bottomSheetTextInput/BottomSheetTextInput.js
index 55c6308..e4cb096 100644
--- a/node_modules/@gorhom/bottom-sheet/lib/commonjs/components/bottomSheetTextInput/BottomSheetTextInput.js
+++ b/node_modules/@gorhom/bottom-sheet/lib/commonjs/components/bottomSheetTextInput/BottomSheetTextInput.js
@@ -39,7 +39,7 @@ const BottomSheetTextInputComponent = /*#__PURE__*/(0, _react.forwardRef)(({
   }, [onFocus, animatedKeyboardState]);
   const handleOnBlur = (0, _react.useCallback)(args => {
     const keyboardState = animatedKeyboardState.get();
-    const currentFocusedInput = (0, _utilities.findNodeHandle)(_reactNative.TextInput.State.currentlyFocusedInput());
+    const currentFocusedInput = _reactNative.TextInput.State?.currentlyFocusedInput ? (0, _utilities.findNodeHandle)(_reactNative.TextInput.State.currentlyFocusedInput()) : null;
 
     /**
      * we need to make sure that we only remove the target
diff --git a/node_modules/@gorhom/bottom-sheet/lib/module/components/bottomSheetTextInput/BottomSheetTextInput.js b/node_modules/@gorhom/bottom-sheet/lib/module/components/bottomSheetTextInput/BottomSheetTextInput.js
index 4be3599..3f005f4 100644
--- a/node_modules/@gorhom/bottom-sheet/lib/module/components/bottomSheetTextInput/BottomSheetTextInput.js
+++ b/node_modules/@gorhom/bottom-sheet/lib/module/components/bottomSheetTextInput/BottomSheetTextInput.js
@@ -34,7 +34,7 @@ const BottomSheetTextInputComponent = /*#__PURE__*/forwardRef(({
   }, [onFocus, animatedKeyboardState]);
   const handleOnBlur = useCallback(args => {
     const keyboardState = animatedKeyboardState.get();
-    const currentFocusedInput = findNodeHandle(RNTextInput.State.currentlyFocusedInput());
+    const currentFocusedInput = RNTextInput.State?.currentlyFocusedInput ? findNodeHandle(RNTextInput.State.currentlyFocusedInput()) : null;
 
     /**
      * we need to make sure that we only remove the target
diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetTextInput/BottomSheetTextInput.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetTextInput/BottomSheetTextInput.tsx
index 861d276..caa481d 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetTextInput/BottomSheetTextInput.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetTextInput/BottomSheetTextInput.tsx
@@ -44,9 +44,9 @@ const BottomSheetTextInputComponent = forwardRef<
   const handleOnBlur = useCallback(
     (args: NativeSyntheticEvent<TextInputFocusEventData>) => {
       const keyboardState = animatedKeyboardState.get();
-      const currentFocusedInput = findNodeHandle(
-        RNTextInput.State.currentlyFocusedInput()
-      );
+      const currentFocusedInput = RNTextInput.State?.currentlyFocusedInput
+        ? findNodeHandle(RNTextInput.State.currentlyFocusedInput())
+        : null;
 
       /**
        * we need to make sure that we only remove the target
